client/my-sites/patterns/components/pattern-gallery/client.tsx

```diff
@@ -15,6 +15,7 @@ import { getTracksPatternType } from 'calypso/my-sites/patterns/lib/get-tracks-p
 import { PatternTypeFilter, type PatternGalleryFC } from 'calypso/my-sites/patterns/types';
 import { useSelector } from 'calypso/state';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import { usePatternsContext } from '../../context';
```

Comment:
We should use an absolute path in order to stay consistent (here and in other components).

Reply:
Thank you - I need to stop relying so heavily VSCode's autofill for these 😆